src/query/storages/fuse/fuse/src/io/locations.rs

```diff
@@ -128,3 +144,21 @@ impl SnapshotLocationCreator for SnapshotVersion {
         }
     }
 }
+
+impl SnapshotLocationCreator for TableSnapshotStatisticsVersion {
+    fn create(&self, id: &Uuid, prefix: impl AsRef<str>) -> String {
+        format!(
+            "{}/{}/{}{}",
+            prefix.as_ref(),
```

Comment:
how about adding a dedicated sud dir for the state of statistics? 
currently,  SnapshotsIO assumes that there are only snapshot files under `_ss` dir.